### ISSUE TITLE
M5StickC Plus Freezing and No Screen after flashing .bin

### ISSUE THREAD

**Issue by TheTest21** [justcallmekoko/ESP32Marauder#291]
I am trying to get marauder working on my M5StickS Plus for hour and nothing is working I have tried using the ota but it shows this. 

```
exec: "cmd": executable file not found in %PATH%

Compilation error: exec: "cmd": executable file not found in %PATH%
```
and when I use the esptool, the M5 just freezes up and when trying to reboot it, the screen just doesn't even turn on.

Thanks in advance.

**Comment by mk333333** [comment]
> You can use espwebtool, download boot loader,partions,boot app, and firmware and just flash them through site

But which ones do you have to download? because it doesn't put which ones are for the M5StickC PLUS!
ESP32 Marauder v4, v6, Kit, Mini?
Flipper Zero WiFi Dev Board
Flipper Zero Multi Board S3?
WiFi Dev Board Pro/LDDB/NodeMCU-32S/ESP32 Wemos D1 Mini?

From which of the 4 groups do you have to download the files?

**Comment by f3shveg** [comment]
try this 
[ESP32Marauder](https://github.com/justcallmekoko/ESP32Marauder/tree/master)/[FlashFiles](https://github.com/justcallmekoko/ESP32Marauder/tree/master/FlashFiles)
/MarauderV4/

**Comment by f3shveg** [comment]
#

> > You can use espwebtool, download boot loader,partions,boot app, and firmware and just flash them through site
> 
> But which ones do you have to download? because it doesn't put which ones are for the M5StickC PLUS! ESP32 Marauder v4, v6, Kit, Mini? Flipper Zero WiFi Dev Board Flipper Zero Multi Board S3? WiFi Dev Board Pro/LDDB/NodeMCU-32S/ESP32 Wemos D1 Mini?
> 
> From which of the 4 groups do you have to download the files?

So everything good ig?

**Comment by TheTest21** [comment]
update well i tried the eswebtool thing and it didn't work but I got is working using the OTA and a raspberry pi 4 but the new issue is that when turned off apparently the .bin file disappears because when turned the screen is not even on.

**Comment by f3shveg** [comment]
I can only give you a guide how to ude espwebtool
1.Burn your m5stickc (and go to [https://esp.huhn.me/](url))
2.Connect m5stick via usb and choose your port 
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/3e61c0b9-674f-4957-85d2-786fb5a6a227)
3.Then you will see this menu
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/ed5b4c98-4d45-4cda-bd96-6893c3e2147e)
4.Download all files from links and insert them in slots like on screenshot
[https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/MarauderV4/esp32_marauder.ino.bootloader.bin](url)
[https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/MarauderV4/esp32_marauder.ino.partitions.bin](url)
[https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/FlipperZeroMultiBoardS3/boot_app0.bin](url)
[https://github.com/justcallmekoko/ESP32Marauder/releases/download/v0.10.8/esp32_marauder_v0_10_8_20230621_m5stickc_plus.bin](url)
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/ac4c1d30-7912-45ae-b652-6d8ec5864460)
5.Click program
If you get errors like this 
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/177bc7d5-6700-43b9-8fac-37382afb6b45)
I reccomend you to burn your m5stick again, i used m5burner for that

**Comment by f3shveg** [comment]
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/9d882022-768f-4dc6-9e1e-dec103e70309)

**Comment by TheTest21** [comment]
Can you explain how you used m5 burner I have tried to burn the .bin using it but I couldn't figure it out. And also did you use espwebtool with the arduino code?

**Comment by f3shveg** [comment]
> Can you explain how you used m5 burner I have tried to burn the .bin using it but I couldn't figure it out. And also did you use espwebtool with the arduino code?

No, i only used bins that were uploaded by @justcallmekoko, i tried to compile it myself but i got 'firsy defined here' error and i gave up.
So, for the burner
1.Download the firmware(ui flow for example
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/82ffbcae-5e6a-466a-8fad-fa7a8641e23f)
2.Press burn(you can skip wifi configuration)
![image](https://github.com/justcallmekoko/ESP32Marauder/assets/58720920/c2669cde-177c-4bfd-a686-eaf0fa2bb0b5)
3.Press erase

**Comment by TheTest21** [comment]
only thing that has worked for me has been using the OTA and Arduino but my m5 has to stay on or else everything erases.

**Comment by mk333333** [comment]
> I can only give you a guide how to use espwebtool 1.Burn your m5stickc (and go to [https://esp.huhn.me/](url)) 2.Connect m5stick via usb and choose your port ![image](https://user-images.githubusercontent.com/58720920/249641809-3e61c0b9-674f-4957-85d2-786fb5a6a227.png) 3.Then you will see this menu ![image](https://user-images.githubusercontent.com/58720920/249641889-ed5b4c98-4d45-4cda-bd96-6893c3e2147e.png) 4.Download all files from links and insert them in slots like on screenshot [https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/MarauderV4/esp32_marauder.ino.bootloader.bin](url) [https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/MarauderV4/esp32_marauder.ino.partitions.bin](url) [https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/FlipperZeroMultiBoardS3/boot_app0.bin](url) [https://github.com/justcallmekoko/ESP32Marauder/releases/download/v0.10.8/esp32_marauder_v0_10_8_20230621_m5stickc_plus.bin](url) ![image](https://user-images.githubusercontent.com/58720920/249642241-ac4c1d30-7912-45ae-b652-6d8ec5864460.png) 5.Click program If you get errors like this ![image](https://user-images.githubusercontent.com/58720920/249642364-177bc7d5-6700-43b9-8fac-37382afb6b45.png) I reccomend you to burn your m5stick again, i used m5burner for that

Following this guide I was able to install it.
The only thing I would add is that when the flash finished installing it would stay frozen without being able to do anything or go with any button. 
You have to unplug it and wait for the battery to run out. When it turns off and you plug it back in. 
it works now!!!!

THANK YOU SO MUCH!

**Comment by f3shveg** [comment]
> > I can only give you a guide how to use espwebtool 1.Burn your m5stickc (and go to [https://esp.huhn.me/](url)) 2.Connect m5stick via usb and choose your port ![image](https://user-images.githubusercontent.com/58720920/249641809-3e61c0b9-674f-4957-85d2-786fb5a6a227.png) 3.Then you will see this menu ![image](https://user-images.githubusercontent.com/58720920/249641889-ed5b4c98-4d45-4cda-bd96-6893c3e2147e.png) 4.Download all files from links and insert them in slots like on screenshot [https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/MarauderV4/esp32_marauder.ino.bootloader.bin](url) [https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/MarauderV4/esp32_marauder.ino.partitions.bin](url) [https://github.com/justcallmekoko/ESP32Marauder/raw/master/FlashFiles/FlipperZeroMultiBoardS3/boot_app0.bin](url) [https://github.com/justcallmekoko/ESP32Marauder/releases/download/v0.10.8/esp32_marauder_v0_10_8_20230621_m5stickc_plus.bin](url) ![image](https://user-images.githubusercontent.com/58720920/249642241-ac4c1d30-7912-45ae-b652-6d8ec5864460.png) 5.Click program If you get errors like this ![image](https://user-images.githubusercontent.com/58720920/249642364-177bc7d5-6700-43b9-8fac-37382afb6b45.png) I reccomend you to burn your m5stick again, i used m5burner for that
> 
> Following this guide I was able to install it. The only thing I would add is that when the flash finished installing it would stay frozen without being able to do anything or go with any button. You have to unplug it and wait for the battery to run out. When it turns off and you plug it back in. it works now!!!!
> 
> THANK YOU SO MUCH!

You don’t have to wait that much,press and hold left button for like 7-8 seconds and then release and press it once

**Comment by TheTest21** [comment]
I fixed it! I was using using the MaruderV4 boot ap instead of the flipper one and also got the cmd issue with these steps:

Check your system's PATH variable: Verify that the PATH environment variable is correctly set up on your computer. The PATH variable should include the directory where the "cmd" executable is located. By default, the "cmd" executable is typically found in the "C:\Windows\System32" directory. You can check the PATH variable by following these steps:

- Open the Start menu and search for "Environment Variables."

- Click on "Edit the system environment variables" to open the System Properties window.

- In the System Properties window, click on the "Environment Variables" button.

- Look for the "Path" variable under the "System variables" section and click on "Edit."

- Ensure that the path to the "C:\Windows\System32" directory is included in the list of paths. If it is missing, you can add it by clicking on "New" and entering the directory path.

by ChatGPT

**Comment by TheTest21** [comment]
I fixed it! I was using using the MaruderV4 boot ap instead of the flipper one and also got the cmd issue with these steps:

Check your system's PATH variable: Verify that the PATH environment variable is correctly set up on your computer. The PATH variable should include the directory where the "cmd" executable is located. By default, the "cmd" executable is typically found in the "C:\Windows\System32" directory. You can check the PATH variable by following these steps:

- Open the Start menu and search for "Environment Variables."

- Click on "Edit the system environment variables" to open the System Properties window.

- In the System Properties window, click on the "Environment Variables" button.

- Look for the "Path" variable under the "System variables" section and click on "Edit."

- Ensure that the path to the "C:\Windows\System32" directory is included in the list of paths. If it is missing, you can add it by clicking on "New" and entering the directory path.

by ChatGPT

**Comment by laranja3** [comment]
Hello.
My device won't turn on.
It has the right firmware.
It has full battery.
This has already happened to me before, when i first installed this firmware(the ESP32Marauder v0.10.8).However, after 30m it turned on and went well.
I tried some other stuff(other frimwares) for fun but then I deleted them from my device.
After it, i tried the M5burner with the title "M5StickCPlus-FactoryTest
official" and it ran smoothly.
Then, i deleted the firmware again.

NOW I'm trying to install the marauder firmware again (from here: https://github.com/justcallmekoko/ESP32Marauder/releases/tag/v0.10.8)
But my device won't turn on!!!!!!!
Could someone explain to me how can i solve this?
